### DISCUSSION TITLE
Meu link do projeto está sendo acusado como erro de segunraça, alguém pode me ajudar por favor??? estava funcionando normal até ontem

### DISCUSSION THREAD

**Discussion by 066luanzin** [community/community#113689] [Code Security]
### Select Topic Area

Bug

### Body

https://066luanzin.github.io/Meus.links/
![Captura de tela 2024-03-22 124508](https://github.com/community/community/assets/159847981/98e4e36a-5f6e-43d7-9d66-1a7db756fb5e)

**Comment by ettaboyle** [comment]
Hi @066luanzin, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
